### ISSUE TITLE
Apk self creation option

### ISSUE THREAD

**Issue by velteyn** [ligi/SurvivalManual#55]
Hello,
Immagine I have your app but a faulty battery i cannot remove. I foun a working phone and I whant to tranfer the whole app to the new one but internet is not available. 
It whoul helpfull to have a self apk creation option that saves the app into sd so one can transfer and install it to the other cell (or into a frend's one). 
Is it possible ?

**Comment by ligi** [comment]
Thanks for the suggestion! I will think about this one ..

**Comment by Peuc** [comment]
That's probably more a task for your applications manager than the application itself. [F-droid](https://f-droid.org/) has a feature to send installed apks to another device, you definitely should give it a try!

**Comment by velteyn** [comment]
yes yes, for sure APP XYZ have the feature a b c d e etc etc , but in an emergency situation and with no internet and without the APP XYZ you are in big problems
